rotkehlchen/db/upgrades/v36_v37.py

```diff
@@ -213,13 +213,25 @@ def _fix_kraken_eth2_events(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _fix_kraken_eth2_events')
 
 
+def remove_ftx_data(write_cursor: 'DBCursor') -> None:
+    """Removes FTX credentials and subaccounts from the DB"""
+    write_cursor.execute(
+        'DELETE FROM user_credentials WHERE location IN (?, ?)',
+        (Location.FTX.serialize_for_db(), Location.FTXUS.serialize_for_db()),
+    )
+    write_cursor.execute(
+        'DELETE FROM user_credentials_mappings WHERE credential_location IN (?, ?)',
+        (Location.FTX.serialize_for_db(), Location.FTXUS.serialize_for_db()),
+    )
+
```

Comment:
you also need to read the `non_syncing_exchanges` setting from the DB and if it has FTX in there, remove it and write it back again.

Add tests for all these things.